api/provider/aws/load_balancer_delete.go

```diff
@@ -74,12 +74,15 @@ func (l *LoadBalancerProvider) Delete(loadBalancerID string) error {
 		}
 	}
 
-	fn := waitUntilSGDeletedFN(l.AWS.EC2, securityGroupName)
-	if err := retry.Retry(fn, retry.WithTimeout(time.Second*30), retry.WithDelay(time.Second)); err != nil {
-		return errors.New(errors.EventualConsistencyError, err)
+	err = nil
```

Comment:
this is a sign that this function is too large and needs to be broken up into smaller chunks